PravegaGroup.json

```diff
@@ -4,8 +4,8 @@
     {
       "id": "/pravega/bkgroup",
       "apps": [
-      { 
-          "id": "/pravega/bkgroup/bookie", 
+      {
```

Comment:
Could you submit these pravega group changes in a separate PR, please? They are unrelated to doc changes.